source/ast/expressions/AssertionExpr.cpp

```diff
@@ -441,6 +536,11 @@ SequenceRepetition::SequenceRepetition(const SequenceRepetitionSyntax& syntax,
 }
 
 SequenceRepetition::AdmitsEmpty SequenceRepetition::admitsEmpty() const {
+    // If min and max delays are both zero, then the sequence accepts only empty matches.
+    if (range.min == 0 &&
+        (!range.max.has_value() || (range.max.has_value() && range.max.value() == 0)))
```

Comment:
This will say that the repetition admits only empty matches when the max is unbounded (i.e. range.max is unset), which I think is not right. It's only when both values are actually zero that it admits only empty matches.

Reply:
fixed